rules/REQUEST-903.9002-WORDPRESS-EXCLUSION-RULES.conf

```diff
@@ -89,7 +89,7 @@ SecRule REQUEST_FILENAME "@endsWith /wp-comments-post.php" \
 #
 
 # Gutenberg
-SecRule REQUEST_FILENAME "@rx ^/wp-json/wp/v[0-9]+/(?:posts|pages)" \
+SecRule REQUEST_FILENAME "@rx /wp\-json/wp/v[0-9]+/(?:posts|pages)" \
```

Comment:
Nitpick: You don't really need to escape `-` outside a class.

Reply:
Removed the escape.